ngx_sssd_info.c

```diff
@@ -632,7 +632,7 @@ static ngx_int_t ngx_http_sssd_info_handler(ngx_http_request_t *r)
             reply = sssd_dbus_message(r, connection, &error,
                 DBUS_SSSD_TIMEOUT, DBUS_SSSD_GET_USER_GROUPS_METHOD, NULL);
 
-            ngx_int_t count;
+            uint count;
```

Comment:
`dbus_message_iter_get_args()` takes a pointer to integer, so I wouldn't call the type decision "allocation bug". Can you elaborate and explain the reasoning in the commit message?

Since the values it holds really shouldn't get into the negatives, I'm OK with the idea of this change as it shouldn't break the functionality. However, could you use the `ngx_uint_t` instead of `uint`, to be consistent with the rest of nginx types?

Reply:
If all dynamic modules, including ngx_sssd_info, get compiled with `--with-debug`, `ngx_array_create()` fails allocate. I haven't found the time to look into it any further, but changing the type `count` to an `unsinged int` seems to work both ways.